src/apps/Makefile

```diff
@@ -0,0 +1,28 @@
+CC = clang
+
+objectify = $(subst .c,.o, $1)
+
+SRC = parser.c
+OBJS = $(call objectify,$(SRC))
+TARGETS = parser
+SANS = -fsanitize=leak,address
+
+INC=-I../../include -I/usr/include/json-c -I/usr/local/include/botan-2/
```

Comment:
You just broke compability with macOS and brew-based botan :(

Reply:
Ok, it was teribbly stuid idea to add this makefile.